pkg/porter/mixins.go

```diff
@@ -62,6 +63,41 @@ func (p *Porter) ListMixins() ([]mixin.Metadata, error) {
 	return mixins, nil
 }
 
+// SearchMixins searches the internal remote mixins list according to the provided options
+func (p *Porter) SearchMixins(opts mixin.SearchOptions) error {
+	box := packr.New("get.porter.sh/porter/pkg/mixin/remote-mixins", "../mixin/remote-mixins")
+	mixinSearcher := mixin.NewSearcher(box)
+
+	remoteMixins, err := mixinSearcher.Search(opts)
+	if err != nil {
+		return err
+	}
+
+	if len(remoteMixins) == 0 && opts.Name != "" {
+		fmt.Fprintf(p.Err, "No mixins found for %s\n", opts.Name)
+		return nil
+	}
+
+	switch opts.Format {
+	case printer.FormatTable:
+		printMixinRow :=
+			func(v interface{}) []interface{} {
+				m, ok := v.(mixin.RemoteMixinListing)
+				if !ok {
+					return nil
+				}
+				return []interface{}{m.Name, m.Description, m.Author, m.SourceURL, m.FeedURL}
+			}
+		return printer.PrintTable(p.Out, remoteMixins, printMixinRow, "Name", "Description", "Author", "Source URL", "Feed URL")
```

Comment:
I'm not sure where "Source URL" came from as a term? Is that following an existing precedent in our CLI that I'm brainfarting on? 😀  

If not, would you mind using "URL" and "Feed URL" to be consistent with `--url` and `--feed-url` that people will be familiar with from the `porter mixins install` command? I'd like to avoid introducing a new term that is an alias for something we already have.